MapboxNavigation.podspec

```diff
@@ -44,9 +44,7 @@ Pod::Spec.new do |s|
   s.module_name = "MapboxNavigation"
 
   s.dependency "MapboxCoreNavigation", "#{s.version.to_s}"
-  s.dependency "MapboxDirections.swift", "~> 0.10.0"
   s.dependency "Mapbox-iOS-SDK", "~> 3.5"
-  s.dependency "OSRMTextInstructions", "~> 0.2.0"
```

Comment:
It doesn't look like we require MapboxCoreNavigation in this podspec though. I think we can add it as a dep or leave as is.